runtime/doc/api.txt

```diff
@@ -810,13 +810,13 @@ nvim_get_current_tabpage()                        *nvim_get_current_tabpage()*
     Gets the current tabpage.
 
     Return: ~
-        Tabpage handle
+        |tab-ID|
 
 nvim_get_current_win()                                *nvim_get_current_win()*
     Gets the current window.
 
     Return: ~
-        Window handle
+        |window-ID|
```

Comment:
This is a useful change. The "handle" jargon isn't used much, and was mostly introduced because of our use of msgpack "extension types", which tried to give a clever object-oriented interface to buffers/windows/tabs.